src/OpenTelemetry/CHANGELOG.md

```diff
@@ -12,6 +12,11 @@
 
 Released 2024-Jun-14
 
+* Exemplars are now supported as a stable feature. Please note that the
```

Comment:
Not passionate enough about this to block it, but to me this feels like maybe the wrong direction.

Thinking we do highlights in our announcements (ex https://cloud-native.slack.com/archives/C01N3BC2W7Q/p1718396414488739) we just don't have this stuff captured somewhere. What if we created a `RELEASENOTES.md` in the root dir to put highlights for major release? It could say something like "This file contains announcements and notes for major features and changes introduced across core components. See individual project CHANGELOGs for details about all changes".

Reply:
It just kind of smells to me. Why this CHANGELOG and not others? Exemplar support is spread around a bunch of CHANGELOGs. Why would users think to look here and not other places? It is also odd it doesn't link to a PR like every other entry.

I also have this problem: https://github.com/open-telemetry/opentelemetry-dotnet/releases/tag/core-1.9.0

I don't like that the 1.9.0 release tag (for example) shows "No notable changes" for most things. You have to dig into all the alpha/beta/rc releases to find what rolls up into 1.9.0. I think having an ANNOUCEMENTS or RELEASENOTES central file would be nice because we could link to it on the releases. We could link to it from main README. We could link to it on the slack announcements. We could link to it from individual CHANGELOGs too (perhaps).

Saw this PR and thought maybe a new central file could be used to solve/help with a lot of this 🤷

Reply:
_Why this CHANGELOG and not others?_

I think we should do it for all changelogs. The changelog entry for the stable release, should summarize/highlight key changes, so it is easy for end users to read it and understand what changed quickly. 
Announcements/Release-notes is very good idea, but we should still duplicate it in changelog's stable release entry too. Announcements can combine the updates from every package, so users can easily what happened in 1.9.0 release.

Reply:
Big giant 👎 from me on this. Unless you are volunteering to do it! 🤣 We just got to a really nice place with our automation, I would hate to [re-]introduce some big manual thing to go and modify all these files with highlights. It's a log of changes, we should keep it at that IMO. But easy enough to put a line in each CHANGELOG (one-time) which says...

```markdown
# Changelog

This file contains individual changes for the OpenTelemetry.X.Y.Z package.
For highlights and announcements covering all components see: [Release Notes](../../RELEASENOTES.md).

## Unreleased
...
```

Seems like it would accomplish the goal no?

Reply:
> Unless you are volunteering to do it! 

No, I am not volunteering to do it. Its a gap I observed. (Like you also said, it is very hard for a user to look at changelog and tell if exemplar is really supported or not, unless they read through full history of beta/alphas).

Don't think we can automate generating highlights, someone has to do it by hand. If doing it in separate file (releasenotes.md), is better with automation, that is totally okay.

Reply:
Ya what I'm thinking is I could update the bot so when you open a "prepare release" PR you can comment/tell it what you want the highlights to be and then it can go and update the file. Should be pretty easy to do. Having to update all the individual files... yuck!

Reply:
@CodeBlanch - This sounds good to me. Forgot about the impact on automation part with this.

Reply:
Are we okay to make this one-time manual thing for Exemplars?